### PR TITLE
Added my new posts for .NET Core development.

### PR DESCRIPTION
**Pull request by m1evankaiser**
- Added post for setting up .NET Core on your environment
- Added post for creating a basic .NET Core REST API

### CODE REVIEW

File: _posts/2017-11-17-setting-up-dotnet-core.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+***
+## Introduction
+
+Today I'm going to go over how to set up your environment for .NET Core development, specifically for REST APIs. First, I'll go over the different tools needed to follow along in this series. 
+
+***
+## Tools
+
+***
```
Comment (m1markcoleman) [inline _posts/2017-11-17-setting-up-dotnet-core.md]: double underline, one under `## Tools` and then another one.

File: _posts/2017-11-17-setting-up-dotnet-core.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+tags:
+  - .NET Core
+  - Setup Scripts
+---
+
+## .NET Core
+
+This is an ongoing series on .NET Core.
+
+1) Setting up .NET Core
```
Comment (m1markcoleman) [inline _posts/2017-11-17-setting-up-dotnet-core.md]: not sure if just when running but the list shows up on one line in the rendered output.

File: _posts/2017-11-21-dotnet-core-rest-api-in-about-30-minutes.md [hunk 3]
```diff
@@ -0,0 +1,587 @@
...
+  }
+  ```
+
+  Try it out by restarting the app, performing the `POST` (so you have something to delete), then have Postman hit the `DELETE` method of the URL: `http://localhost:5000/api/airplanes/12345`. Now if you perform a `GET` for that record, you should receive a `204 - No Content`.
+
+  ***
+  :thought_balloon:
+
+  Based on [good REST patterns](https://restpatterns.mindtouch.us/HTTP_Status_Codes), `204 - No Content` is really not want we want to see for the `GET` of a missing record. We really want to see `404 - Not Found`, but we'll have to modify it to return an `IActionResult`, which we will go over in a later post.
+  ***
```
Comment (m1markcoleman) [inline _posts/2017-11-21-dotnet-core-rest-api-in-about-30-minutes.md]: in the output i see `posts. ***`

File: _posts/2017-11-21-dotnet-core-rest-api-in-about-30-minutes.md [hunk 4]
```diff
@@ -0,0 +1,587 @@
...
+  :question:
+
+  You'll probably be thinking, "Didn't we specifically make the empty constructor and all the properties of Airplane private, so calling code would have to use the constructor? How did we create it without doing that?". Well yes, normally we wouldn't be asking for an actual EF entity in the request, or returning it back to the client. Usually we would use Request/Response "ViewModels" for the API endpoint layer. 
+  
+  However, for the sake of simplicity in the initial setup, we'll just use the entity. The .NET Core default JSON serializer allows private properties, so it will work for now. In future posts I will go over using a Service layer, containing "ViewModels", to prevent the client from being able to control the entity's state.
+  ***
+
+***
+* GET (one)
```
Comment (m1markcoleman) [inline _posts/2017-11-21-dotnet-core-rest-api-in-about-30-minutes.md]: having this in a list pushes the style to the right a bit in the output, would having `#### GET (one) allow for the breakup that is needed but not have the indent that causes the style to get off?

**Review by m1markcoleman — COMMENTED** [review]
Excellent post, just noticed some things when running the post inside of the site.  I asked @m1anitamcallister to give a read as well.

File: _posts/2017-11-21-dotnet-core-rest-api-in-about-30-minutes.md (hunk 3 above)
Comment (m1evankaiser) [reply _posts/2017-11-21-dotnet-core-rest-api-in-about-30-minutes.md]: Wow, I wonder why it's formatting the markdown so much differently than both VS Code Preview and TFS Preview... I'll have to just go through again and fix all these little formatting issues.

File: _posts/2017-11-21-dotnet-core-rest-api-in-about-30-minutes.md (hunk 4 above)
Comment (m1evankaiser) [reply _posts/2017-11-21-dotnet-core-rest-api-in-about-30-minutes.md]: That was actually intentional (to be a list under "More API Methods"), but as with some of the other things, it looked a lot better in VS Code Preview ☹️

Comment (m1markcoleman) [reply _posts/2017-11-21-dotnet-core-rest-api-in-about-30-minutes.md]: Yeah it is so different, the m ⬇️ preview it is spot on but when viewed in the site things get out of whack.

**Review by m1markcoleman — APPROVED** [review]
looks great, let me check with @m1anitamcallister to see if she wants to review before we merge into `master`

**Review by m1anitamcallister — APPROVED** [review]
Looks great.

**Pull request merged**